### ISSUE TITLE
Request: Add "Patterns of use" section for vocab subjects

### ISSUE THREAD

**Issue by jleskovar** [davidsansome/tsurukame#546]
Hi there,

The WaniKani website has recently added support for a "Patterns of Use" section for vocabulary subjects (see screenshot below). This would be an awesome addition, to sit below the context sentences perhaps:

![Screen Shot 2022-02-07 at 8 51 33 am](https://user-images.githubusercontent.com/232534/152702885-e133db5c-9796-4992-8be0-acd2e9967181.png)

**Comment by jleskovar-tyro** [comment]
Good point - raised a question over on the WK forums, if anyone's keen to throw in their support :) 
https://community.wanikani.com/t/exposing-patterns-of-use-data-for-vocabulary-subjects/55680
